shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -643,7 +643,10 @@ export default {
         <span v-if="isHarvesterCluster">
           <a
             role="button"
+            :aria-label="displayProvider"
             @click="goToHarvesterCluster"
+            @keyup.space="goToHarvesterCluster"
+            @keyup.enter="goToHarvesterCluster"
           >
```

Comment:
Do we get any benefit out of preserving the anchor element? I still think that it makes more sense to use a button in these cases, we can use `<button class="btn role-link"` and that should produce a similar effect.